lib/mongo/protocol/query.rb

```diff
@@ -174,7 +174,7 @@ def determine_limit
       FLAGS = [
         :reserved,
         :tailable_cursor,
-        :slave_ok,
+        :secondary_ok,
```

Comment:
We generally consider the operation and protocol layer private. If so the flag list can be mutated at will but it also should be marked @api private given that the class itself is "semiprivate".

Have you considered whether users may legitimately use this flag list for anything?

Reply:
Marked with `@api private`